### DISCUSSION TITLE
After rerunning the script I'm getting error in payment ,  scripting is getting success but it's not working

### DISCUSSION THREAD

**Discussion by Kurra432** [learndevopsonline/learndevopsonline#1963] [Q&A]
![image](https://github.com/user-attachments/assets/dafa3714-ed0e-4ef6-b149-52f35d0618c4)

![image](https://github.com/user-attachments/assets/e29038b3-4d3b-4ef9-9021-ab0734d46f8f)

**Comment by chaithanya1812** [comment]
Hi @Kurra432 
Everything is fine.. just reverify once more

**Reply by Kurra432** [reply]
Hi @chaithanya1812  I verified multiple times but where I'm getting error is not getting 
![image](https://github.com/user-attachments/assets/41f14abf-c777-4c27-8b75-4bd76035406f)

**Reply by Kurra432** [reply]
Frontend Public IP - 54.152.85.206

**Reply by chaithanya1812** [reply]
Hi @Kurra432 
## Error present in payment.. so it's easy one fix it ,I am expecting  from your side 
![image](https://github.com/user-attachments/assets/3a3695d6-dae2-429b-aa99-c051cf569b8e)

**Reply by Kurra432** [reply]
@chaithanya1812 I'm not getting where I have missed

**Reply by chaithanya1812** [reply]
Hi @Kurra432 

![image](https://github.com/user-attachments/assets/14acbe62-28cd-40ce-bb49-b63d7fef82d7)


# look at sed command what I changed??

![image](https://github.com/user-attachments/assets/27b617ba-48c6-4db2-945c-f820be973d66)

**Reply by Kurra432** [reply]
Ok , Got it I missed the double quotes

**Comment by Kurra432** [comment]
Issue resolved
